scripts/process_nf.py

```diff
@@ -354,25 +354,11 @@
 # %% CHECKING OUTPUT
 #==============================================================================
 
-if matplot:
-    if output_plot_check:
-
-        plt.figure(1)
-        plt.imshow(confidence_map[0])
-        plt.title('Bottom Layer Confidence')
 
-        plt.figure(2)
-        plt.imshow(confidence_map[-1])
-        plt.title('Bottom Layer Confidence')
+mat = Material(name=mat_name, material_file=mat_file, dmin=valWUnit(
+    'lp', 'length',  0.05, 'nm'), kev=valWUnit('kev', 'energy', beam_energy, 'keV'))
```

Comment:
Maybe this should be moved into the `if output_plot_check:` block since it is only used there?